script/Arcueid/Day 10/10_00_ARC10_3_3.txt

```diff
@@ -463,32 +484,37 @@ There would be something missing in the process before acting, right? Or rather,
 [sha:158e33fd12c6f7b7393e0be60fcb3c2d9649c1f5]{
 -- Page 279, Offset 24968.
 -- 感心したような、呆れたような、そんな相づちを打って、アルクェイドは噴水に視線を戻した。
-Giving off an agreeable response of what seems like admiration or astonishment, Arcueid turns her sight back to the water fountain.
+With a soft murmur of either admiration or astonishment,
+ Arcueid turns her attention back toward the fountain.
 }
 [sha:8bc2a77fe5f4d3589c91b25b5de76cd9c1196284]{
 -- Page 280, Offset 24969.
 -- 「ねえ志貴。関係のない事を聞くけど、志貴にもああいう時があったの？」
-"Hey, Shiki. This is completely unrelated, but do you have times like that as well?"
+"Hey, Shiki. This is completely unrelated, but did you use to be like that too?"
 }
 [sha:0dc893bb53707b8b89a5eb784ab3ff6c8a2c1597]{
 -- Page 281, Offset 24970.
 -- 本当に突然、まったく関係のない事を聞いてくる。
-In a truly sudden fashion, Arcueid asks me a completely unrelated question.
+Abruptly switching gears, Arcueid hits me with a new question out of nowhere.
 }
 [sha:55bbd45fb217f873230a90bc9700f11817d43ccc]{
 -- Page 281, Offset 24971.
 -- アルクェイドの視線の先には遊んでいる子供たちがいるから、彼女の聞きたい事はそういう事なんだろう。
-There are children playing around in her line of sight, so that must be why she's asking me that.
+She's watching some children playing around in front of her,
+ so presumably that's what she's asking about.
 }
 [sha:62859116c54074465f32a293310815bdebcc3413]{
 -- Page 282, Offset 24972.
 -- 「さあ、どうかな。たしかに子供のころは遊びまわってたけど、それは屋敷の中だけの話でさ。こういう公園で遊んだ事は少なかった」
-"Who knows? It's true that I played around when I was a kid, but it was only inside the mansion. I never really played around in a park like this much."
+"Hmm, hard to say.
```

Comment:
"hard to say" is a little less flippant than "who knows" so i say revert

Reply:
Hmm, I thought this was more musing than flippant. I'll go back and re-listen.

Reply:
Hm yeah it sounds more considered to me

Reply:
alright